collective/elasticsearch/mapping.py

```diff
@@ -63,7 +63,9 @@ def __call__(self):
         for name in self.catalog.indexes.keys():
             index = getIndex(self.catalog, name)
             if index is not None:
-                properties[name] = index.create_mapping(name)
+                # prevent create default mapping analyzers
+                if name not in properties:
```

Comment:
Why is this necessary? Is it necessary for ES7 or does ES6 have this problem too?

Reply:
It's a problem with occur in ES6 and ES7. 
To create mapping with the fields analyzed and tokenized. The code currently ignores the mapping configuring and apply without configuration.

Reply:
ok

Reply:
I took a look and I still don't understand the need for this test. The current code overrides the default configuration by index type configuration. Why use the default configuration and not the index type configuration? Can you give an example of a problem index?